### PR TITLE
Exlude some rows feature

### PR DESCRIPTION
**Pull request by vhessam**
## Description

I added a feature to the search query logic

## Motivation and context

I had a requirement to not get some indexable records (my models which use Indexable). In my scenario think about a blog system which my model (the post model) has an is_published column. If this column is false is must not get anything search result about that post. I want it to be handled when I make query not after I get results from search and pruning it.

## How has this been tested?

Please describe in detail how you tested your changes.

Include details of your testing environment, and the tests you ran to
see how your change affects other areas of the code, etc.

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

Go over all the following points, and put an `x` in all the boxes that apply.

Please, please, please, don't send your pull request until all of the boxes are ticked. Once your pull request is created, it will trigger a build on our [continuous integration](http://www.phptherightway.com/#continuous-integration) server to make sure your [tests and code style pass](https://help.github.com/articles/about-required-status-checks/).

- [x] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [x] My pull request addresses exactly one patch/feature.
- [x] I have created a branch for this patch/feature.
- [x] Each individual commit in the pull request is meaningful.
- [ ] I have added tests to cover my changes.
- [x] If my change requires a change to the documentation, I have updated it accordingly.

If you're unsure about any of these, don't hesitate to ask. We're here to help!

### CODE REVIEW

**Comment by vhessam** [comment]
Hello,
I cant understand the error for CI build process!

**Comment by JaZo** [comment]
@vhessam the CI failure is caused by code style errors. Please run `composer fix-style` and amend the results in your commit.

**Comment by vhessam** [comment]
OK I'll do it

File: README.md [hunk 1]
```diff
@@ -106,6 +106,11 @@ Results on ``title`` or ``content`` are weighted in the results. Search result s
 
 Enable wildcard after words. So when searching for for example  ``car`` it will also match ``carbon``. 
 
+### exclude_feature_enabled
+This feature excludes some rows from being  returned. Enable this when you have a flag in your model which determines whether this record must be returned in search queries or not. By default this feature is disabled.
+### exclude_records_column_name
```
Comment (bbrala) [inline README.md:111]: Missing newline

File: README.md [hunk 2]
```diff
@@ -106,6 +106,11 @@ Results on ``title`` or ``content`` are weighted in the results. Search result s
 
 Enable wildcard after words. So when searching for for example  ``car`` it will also match ``carbon``. 
 
+### exclude_feature_enabled
+This feature excludes some rows from being  returned. Enable this when you have a flag in your model which determines whether this record must be returned in search queries or not. By default this feature is disabled.
```
Comment (bbrala) [inline README.md:110]: Double space :)

Comment (bbrala) [reply README.md:110]: Also a small example would be nice i think since.

Comment (vhessam) [reply README.md:110]: OK I'll add and example too

**Comment by vhessam** [comment]
I updated readme file with an example. Please check it out .

**Comment by bbrala** [comment]
Thanks! Merged :)

**Comment by vhessam** [comment]
@bbrala many thanks ;)

**Pull request merged**